modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-configuring-the-cluster-for-a-disconnected-environment_{context}"]
+= Configuring a managed cluster for a disconnected environment
+
+After you have completed the preceding procedure, follow these steps to configure the managed cluster for a disconnected environment.
+
+.Prerequisites
+
+* A disconnected installation of Red Hat Advanced Cluster Management (RHACM) 2.3.
+
+* Host the `rootfs` and `iso` images on an HTTPD server.
+
+.Procedure
+
+. Create a `ConfigMap` containing the mirror registry config:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: assisted-installer-mirror-config
+  namespace: assisted-installer
+  labels:
+    app: assisted-service
+data:
+  ca-bundle.crt: <certificate> <1>
+  registries.conf: |  <2>
+    unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
+
+    [[registry]]
+      location = <mirror-registry-url>  <3>
+      insecure = false
+      mirror-by-digest-only = true
+----
+<1> `certificate` is the mirror registry’s certificate used when creating the mirror registry.
+<2> `registry-config` is the configuration for the mirror registry.
+<3> `mirror-registry-url` is the URL of the mirror registry.
++
+This is an example of what the AgentServiceConfig might look like:
```

Comment:
Please confirm that this section is as you intend - it doesn't appear to be connected to the previous step.